include/begin_code.h

```diff
@@ -36,6 +36,8 @@
 #ifndef SDL_DEPRECATED
 #  if defined(__GNUC__) && (__GNUC__ >= 4)  /* technically, this arrived in gcc 3.1, but oh well. */
 #    define SDL_DEPRECATED __attribute__((deprecated))
+#  elif defined(_MSC_VER)
+#    define SDL_DEPRECATED __declspec(deprecated)
```

Comment:
I don't know the minimum VC version that supports this.
I assume MSVS 2010 supports this?

Reply:
I think this change can also go in SDL3.
Ok?

Reply:
Yep!

Reply:
I cherry-picked it at 917e036f6fdefee5d60c0adcde3476cefbcbbf8d